geode-junit/src/main/java/org/apache/geode/test/junit/rules/gfsh/DebuggableCommand.java

```diff
@@ -25,7 +27,7 @@ public DebuggableCommand(String command) {
   public DebuggableCommand(String command, int debugPort) {
 
     if (command.startsWith("start locator")) {
-      command += " --J=-Denable-experimental-cluster-management-service=true ";
+      command += " --J=-D" + ClusterManagementService.FEATURE_FLAG + "true ";
```

Comment:
This is missing an `=` (equals) sign.... `"=true"

Reply:
Thanks!